src/ResourceManager/Storage/Commands.Management.Storage/Az.Storage.psd1

```diff
@@ -182,10 +182,13 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = 'Update release with latest service features and serialization fixes'
+        ReleaseNotes = '* Support failover Storage account
+    - Invoke-AzureRmStorageAccountFailover
+* Support Get last sync time of Storage account
+    - Get-AzureRmStorageAccountLastSyncTime'
 
         # Prerelease string of this module
-        # Prerelease = ''
+        Prerelease = 'preview'
```

Comment:
@markcowl I thought for `0.x.x` versions of modules we weren't adding the `preview` suffix for preview modules -- I thought we were only adding the suffix for stable modules.

Reply:
this is a preview version of a stable module